test/mexpr/float-test.mc

```diff
@@ -53,8 +53,7 @@ utest 0.25 with 7.5 using neg geqf in
 utest 0.25 with 0.25 using geqf in
 utest 2.2 with 2.2 using eqf in          -- equal (float)
 utest 2.2 with 2.3 using neg eqf in
-utest subf 3.0 5.0 with negf 2.0
-      using eqf in
+utest subf 3.0 5.0 with negf 2.0 using eqf in
```

Comment:
I'm assuming changes in this file are related to wasm changes? How come there are changes for the constants? (I'm assuming the commented out `stringIsFloat` are due to lack of intrinsics or some such)

Reply:
Yeah, this is old stuff. I removed it.